app/status.go

```diff
@@ -453,6 +456,24 @@ func (a *App) GetCustomStatus(userID string) (*model.CustomStatus, *model.AppErr
 	return user.GetCustomStatus(), nil
 }
 
+func (a *App) GetRecentCustomStatuses(userID string) ([]*model.CustomStatus, error) {
+	pref, err := a.GetPreferenceByCategoryAndNameForUser(userID, model.PreferenceCategoryCustomStatus, model.PreferenceNameRecentCustomStatuses)
+	if err != nil {
+		return nil, err
+	}
+
+	var existingRCS []*model.CustomStatus
+	if jsonErr := json.Unmarshal([]byte(pref.Value), &existingRCS); jsonErr != nil {
+		return nil, jsonErr
+	}
+
+	if err != nil {
+		return nil, err
+	}
```

Comment:
We need to remove this, it's already handled in `L:461`

Reply:
removed